backend/src/apiserver/server/report_server.go

```diff
@@ -136,6 +158,20 @@ func validateReportScheduledWorkflowRequest(swfManifest string) (*util.Scheduled
 	return swf, nil
 }
 
+func (s *ReportServer) canAccessWorkflow(ctx context.Context, executionName string, resourceAttributes *authorizationv1.ResourceAttributes) error {
+	if !common.IsMultiUserMode() {
```

Comment:
Having in mind the above comment - IMO, there is no need to do this check `IsMultiUser`